Test/lit.site.cfg

```diff
@@ -103,7 +103,8 @@ if len(boogieParams) > 0:
 # Inform user what executable is being used
 lit_config.note('Using Boogie: {}'.format(boogieExecutable))
 
-config.substitutions.append(('%boogie', boogieExecutable))
+config.substitutions.append(('%boogie', boogieExecutable + ' /vcsCores:2'))
+config.substitutions.append(('%sequential-boogie', boogieExecutable))
```

Comment:
I would prefer if you used boogie for the sequential version and parallel-boogie for the parallel version.

Reply:
I thought letting `boogie` be `parallel-boogie` would be nicer since it implies that parallel is the default, leading to more tests using the parallel feature. I can also imagine that we don't want that to be the default, but in that case I wonder which tests should we pick to use `parallel-boogie`, certainly not as many of them as we can, as I've done in this PR, right?

The downside of tests using `parallel-boogie` is that if some of the debug options are used the test can be unstable, although I'm guessing this is detected early on.

Reply:
@RustanLeino :  Please chime in.